### ISSUE TITLE
[docs] nodejs+express: "getting started" guide

### ISSUE THREAD

**Issue by J-Vaughan** [Carleton-Web-Dev-Club/club-site#30]
I think we should have a guide for both like a permanent installation, as well as a more temporary environment (since some/many of CWDC members may only be here for the credit).

**Comment by saihaj** [comment]
I agree. I was also thinking about writing docs for project management so that everyone understands the issue tracker, PR reviews and all that good stuff
